test/blackbox-tests/test-cases/pkg/opam-package-with-subst.t

```diff
@@ -41,6 +32,5 @@ The lockfile should contain the substitute action.
   > EOF
 
 The file foo.ml should have been built:
-
```

Comment:
Intentional space. Could you review all the changes and remove all the unnecessary whitespace tweaks?